### ISSUE TITLE
Issue running training

### ISSUE THREAD

**Issue by sunmart** [murthylab/sleap#616]
Hi,

We've been trying to set up SLEAP in the lab and have been following protocols from other labs in the department that already have SLEAP running. We keep running this issue during training where the program never proceeds beyond a single epoch and we don't see anything on the loss function graph. Do you think this is an issue that has to do with the number of frames we labeled (too few? Based on the tutorial, I would expect to see something, even if it isn't accurate after labeling about 20 frames) or would you say it has something to do with the GPU? I unselected visualization to see if that would help but it didn't. This is approximately where the program gets stuck each time - 

2021-12-21 11:43:32.740352: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x21c6bf59bf0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2021-12-21 11:43:32.747526: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 3080, Compute Capability 8.6
INFO:sleap.nn.training:Loaded test example. [88.071s]
INFO:sleap.nn.training:  Input shape: (736, 1280, 1)
INFO:sleap.nn.training:Created Keras model.
INFO:sleap.nn.training:  Backbone: UNet(stacks=1, filters=32, filters_rate=1.5, kernel_size=3, stem_kernel_size=7, convs_per_block=2, stem_blocks=0, down_blocks=5, middle_block=True, up_blocks=3, up_interpolate=True, block_contraction=False)
INFO:sleap.nn.training:  Max stride: 32
INFO:sleap.nn.training:  Parameters: 2,936,102
INFO:sleap.nn.training:  Heads:
INFO:sleap.nn.training:    [0] = SingleInstanceConfmapsHead(part_names=['Nose', 'Head Center', 'Neck', 'end-back', 'Tail Base', 'Tail Tip', 'mid-back'], sigma=5.0, output_stride=4, loss_weight=1.0)
INFO:sleap.nn.training:  Outputs:
INFO:sleap.nn.training:    [0] = Tensor("SingleInstanceConfmapsHead_0/BiasAdd:0", shape=(None, 184, 320, 7), dtype=float32)
INFO:sleap.nn.training:Setting up data pipelines...
INFO:sleap.nn.training:Training set: n = 18
INFO:sleap.nn.training:Validation set: n = 2
INFO:sleap.nn.training:Setting up optimization...
INFO:sleap.nn.training:  Learning rate schedule: LearningRateScheduleConfig(reduce_on_plateau=True, reduction_factor=0.5, plateau_min_delta=1e-06, plateau_patience=5, plateau_cooldown=3, min_learning_rate=1e-08)
INFO:sleap.nn.training:  Early stopping: EarlyStoppingConfig(stop_training_on_plateau=True, plateau_min_delta=1e-06, plateau_patience=10)
INFO:sleap.nn.training:Setting up outputs...
INFO:sleap.nn.callbacks:Training controller subscribed to: tcp://127.0.0.1:9000 (topic: )
INFO:sleap.nn.training:  ZMQ controller subcribed to: tcp://127.0.0.1:9000
INFO:sleap.nn.callbacks:Progress reporter publishing on: tcp://127.0.0.1:9001 for: not_set
INFO:sleap.nn.training:  ZMQ progress reporter publish on: tcp://127.0.0.1:9001
INFO:sleap.nn.training:Created run path: Z:/Behavior videos/PMV Oxtr fl/Set 1 - Jax/112519 Test1 Pups/191125 1M side_ test_Sunaina_20211219\models\211221_114200.single_instance.n=20
INFO:sleap.nn.training:Setting up visualization...
INFO:sleap.nn.training:Finished trainer set up. [89.6s]
INFO:sleap.nn.training:Creating tf.data.Datasets for training data generation...
INFO:sleap.nn.training:Finished creating training datasets. [3.3s]
INFO:sleap.nn.training:Starting training loop...
Epoch 1/200
2021-12-21 11:43:41.197983: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library cudnn64_7.dll
C:\Users\bambahmukkulab\.conda\envs\sleap\lib\site-packages\shiboken2\files.dir\shibokensupport\signature\parser.py:97: FutureWarning: split() requires a non-empty pattern match.
  return [x.strip() for x in split(argstr) if x.strip() not in ("", ",")]
  
  Thanks for your help!

**Comment by talmo** [comment]
Hi @sunmart!

What version of SLEAP are you running?

With newer GPUs, we need to use a newer version of TensorFlow. We currently have a pre-release version of SLEAP that supports this -- check out: https://github.com/murthylab/sleap/issues/454#issuecomment-977705621

Cheers,

Talmo
